kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/TestUtils.scala

```diff
@@ -68,6 +68,11 @@ trait TestUtils extends Assertions {
     }
   }
 
+  def latestSnapshot(path: String): Snapshot = {
```

Comment:
@scottsand-db didn't we discuss explicitly not using this as a test API?

Reply:
i revert my opinion; im fine with this; thanks!